src/app/components/surveillance/complaints/complaint-view.jsx

```diff
@@ -85,14 +85,12 @@ function ChplComplaintView(props) {
               <Typography variant="subtitle1">General Info</Typography>
               <Typography variant="subtitle2">
                 Received Date:
-              </Typography>
```

Comment:
This now looks like the Typography block spans the Received Date all the way to the Closed Date, and I don't think that's good
![image](https://user-images.githubusercontent.com/909012/127531527-b5f19d6e-b096-4361-81b2-50c43946f715.png)